docs/en/integrations/data-ingestion/data-formats/json.md

```diff
@@ -0,0 +1,576 @@
+---
+sidebar_label: JSON
+sidebar_position: 2
+slug: /en/integrations/data-formats/json
+---
+
+# Importing and exporting JSON data in ClickHouse
+
+JSON is a popular format for exchanging data between different layers of modern applications. ClickHouse provides many tuning options to support almost any form of JSON data.
+
```

Comment:
:::tip JSON tutorial
For a step by step tutorial with a large JSON dataset, please see [Loading JSON in 5 steps](/docs/en/guides/developer/working-with-json/json-load-data.md).
:::

Reply:
Added